.github/workflows/tests.yaml

```diff
@@ -80,6 +80,8 @@ jobs:
         file: ./coverage.xml
         fail_ci_if_error: false
   notify:
+    permissions:
+      contents: none
```

Comment:
Should we be setting `statuses: read` here?

also, is there a reason you aren't setting the explicit `contents: read` permission for the tests job?